jenkins/dockerized-e2e-runner.sh

```diff
@@ -74,7 +74,7 @@ fi
 CONTAINER_NAME="${JOB_NAME}-${BUILD_NUMBER}"
 
 echo "Starting..."
-timeout -s KILL ${DOCKER_TIMEOUT:-60m} docker run --rm \
+docker run --rm \
```

Comment:
I am not super confident this will work the way you're expecting... bootstrap.py is going to send this script a kill signal, which it will ignore because it is waiting for the docker run command.

So I would like an explanation of how the container cleanup code either a) still works or else b) is no longer needed.

I am not super confident we should do this outside of migrating these jobs to json. Specifically I would expect `kubernetes_e2e.py` to trap the kill signal and then delete the container, like we do right now in bash.